go.spec

```diff
@@ -0,0 +1,13 @@
+### RPM external go 1.13.8
+
```

Comment:
Add no compiler to avoid gcc dependency.
```
## NOCOMPILER
```

Reply:
done